frontend/src/components/authentication/login.tsx

```diff
@@ -92,6 +94,9 @@ export default function LoginPage({
     try {
       const user = await Auth.signIn(username, password);
       setUser(user.userSub);
+      if (!user.userConfirmed) {
```

Comment:
- `user.userConfirmed` is showing as null.  (This is because the user result of calling `Auth.signIn()` returns a `CognitoUser` type, which is not the same an an `IsSignUpResult` type)  
 - As a result, `setUser(user.userSub)` is setting user to undefined     
```
// for type IsSignUpResult
{
    user: CognitoUser;
    userConfirmed: boolean;
    userSub: string;
} 
```

```
// for type CognitoUser
{
	Session: null
	attributes: {sub: '171f3454-25a4-4990-a44b-b42225291ffd', email_verified: false, name: 'Marissa Darnell', phone_number_verified: false, phone_number: '+11234567890', …}
	authenticationFlowType: "USER_SRP_AUTH"
	client: Client {endpoint: 'https://cognito-idp.us-west-2.amazonaws.com/', fetchOptions: {…}}
	keyPrefix: "CognitoIdentityServiceProvider.56nsvt3lp1gmb3ou6t63ldj72m"
	pool: CognitoUserPool {userPoolId: 'us-west-2_GsGspX3dl', clientId: '56nsvt3lp1gmb3ou6t63ldj72m', client: Client, advancedSecurityDataCollectionFlag: true, storage: Storage, …}
	preferredMFA: "NOMFA"
	signInUserSession: CognitoUserSession {idToken: CognitoIdToken, refreshToken: CognitoRefreshToken, accessToken: CognitoAccessToken, clockDrift: 0}
	storage: Storage {CognitoIdentityServiceProvider.56nsvt3lp1gmb3ou6t63ldj72m.171f3454-25a4-4990-a44b-b42225291ffd.refreshToken: 'eyJjdHkiOiJKV1QiLCJlbmMiOiJBMjU2R0NNIiwiYWxnIjoiUl…tLQWoEP7fYGSP_G6CmHLkxbViw.VpwWUITxcMCiSWmOcP8ZWg', CognitoIdentityServiceProvider.56nsvt3lp1gmb3ou6t63ldj72m.05f72b57-7a48-4f96-91be-e1887de74fb8.refreshToken: 'eyJjdHkiOiJKV1QiLCJlbmMiOiJBMjU2R0NNIiwiYWxnIjoiUl…W_wM-NY9p77FtCRCSYcXWArbHw.3jAwL9z8FAMXxHyLty54ew', CognitoIdentityServiceProvider.56nsvt3lp1gmb3ou6t63ldj72m.LastAuthUser: '171f3454-25a4-4990-a44b-b42225291ffd', CognitoIdentityServiceProvider.56nsvt3lp1gmb3ou6t63ldj72m.171f3454-25a4-4990-a44b-b42225291ffd.clockDrift: '0', CognitoIdentityServiceProvider.56nsvt3lp1gmb3ou6t63ldj72m.60b31006-e58c-4da8-b0ac-3fbd73a9d7bc.idToken: 'eyJraWQiOiJ1Q3BlRk9ETDUyZDJaOUZYZlo0U2doOE5lU0tsV2…kGQUmBIW_H5E1rcFZLaeO9DbyofkEZu0GRhdjuVs6QmaA2knQ', …}
	userDataKey: "CognitoIdentityServiceProvider.56nsvt3lp1gmb3ou6t63ldj72m.171f3454-25a4-4990-a44b-b42225291ffd.userData"
	username: "171f3454-25a4-4990-a44b-b42225291ffd"
}
```

 - Additionally, a user who is already confirmed is getting redirected to the `/confirm-email` page. 
 - Also, a user who **not confirmed** does not get redirected, I just see a an error in the console, noting that the user is not confirmed  
![image](https://user-images.githubusercontent.com/63181280/164845851-e9943bb9-ac03-401b-a2e3-39bd3fdd1ffe.png)